### PR TITLE
Complete the stitch rename

### PR DESCRIPTION
**Pull request by kayousterhout**

### CODE REVIEW

**Comment by kayousterhout** [comment]
BTW one remaining question here is whether bindings.js should be moved to a dedicated JS folder -- I think there's an argument to be made for that, but I'd like to do that separately from this PR.

File: cli/command/run_test.go [hunk 1]
```diff
@@ -222,7 +222,7 @@ func TestRunFlags(t *testing.T) {
 	t.Parallel()
 
 	expStitch := "blueprint"
-	checkRunParsing(t, []string{"-stitch", expStitch}, Run{blueprint: expStitch}, nil)
```
Comment (kklin) [inline cli/command/run_test.go]: Shouldn't it be that the previous commit should rename this `-stitch` to `-blueprint`?

Also, the line you added is the same as the next line, but that's moot if you change the test to use `-blueprint`.

Comment (kayousterhout) [reply cli/command/run_test.go]: Oops yes sorry misunderstood this! Fixed

**Comment by kayousterhout** [comment]
Thanks for looking at this Kevin!

**Pull request merged**